### DISCUSSION TITLE
How to escape the percent symbol in filter strings?

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#602] [General]
Dear Community,

How does one filter for strings with a percent symbol (the wildcard symbol) in ArcadeDB's SQL? In classic SQL one woud use the `ESCAPE` keyword, ie: `SELECT FROM type WHERE property LIKE 'x \%' ESCAPE '\'`; this keyword seems not to be exported. I also tried to use `\%` without the `ESCAPE` statement, which also yields an error. Did I overlook something?

Thank you for your help

**Comment by lvca** [comment]
In order to escape characters, using the backslash`\` should be enough. If it doesn't work could you please open an issue with a test case?

**Comment by gramian** [comment]
See #603 .
